tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -321,6 +319,75 @@ describe(__filename, () => {
     expect(link).toHaveProp('to', '/addon/chill-out/privacy/');
   });
 
+  it('links to version history if add-on is extension', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_EXTENSION,
+    });
+
+    const root = render({ addon });
+    const history = root.find('.AddonMoreInfo-version-history');
+
+    expect(history).toHaveProp('term', 'Version History');
+    expect(history.find(Link)).toHaveProp(
+      'to',
+      `/addon/${addon.slug}/versions/`,
+    );
+  });
+
+  it('adds a `src` query parameter to the version history link if available in the location', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_EXTENSION,
+    });
+    const src = 'some-src';
+
+    const root = render({
+      addon,
+      location: createFakeLocation({ query: { src } }),
+    });
+
+    const history = root.find('.AddonMoreInfo-version-history');
+
+    expect(history).toHaveProp('term', 'Version History');
+    expect(history.find(Link)).toHaveProp(
+      'to',
+      `/addon/${addon.slug}/versions/?src=${src}`,
+    );
+  });
+
+  it('links to version history if add-on is a dictionary', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_DICT,
+    });
+
+    const root = render({ addon });
+    const history = root.find('.AddonMoreInfo-version-history');
+
+    expect(history).toHaveProp('term', 'Version History');
+    expect(history.find(Link)).toHaveProp(
+      'to',
+      `/addon/${addon.slug}/versions/`,
+    );
+  });
+
+  it('links to version history if add-on is a language pack', () => {
+    const addon = createInternalAddon({
+      ...fakeAddon,
+      type: ADDON_TYPE_LANG,
+    });
+
+    const root = render({ addon });
+    const history = root.find('.AddonMoreInfo-version-history');
+
+    expect(history).toHaveProp('term', 'Version History');
+    expect(history.find(Link)).toHaveProp(
+      'to',
+      `/addon/${addon.slug}/versions/`,
+    );
+  });
+
```

Comment:
Could you please move these test cases where they were before? That would reduce the size of the diff and make it easier to review. Thanks!

Reply:
Yes, I totally understand. I'll move it back to where it was.